.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,259 @@
+      return array(
+        3 => 1,
+        4 => 1,
+        5 => 1,
+        6 => 1,
+        7 => 1,
+        8 => 1,
+      );
+    }
+
+  }
+  ```
+  
+</details><br>  
+
+Also note the class name convention. The method `getWarningList()` MUST return an array of line numbers indicating warnings (when running `phpcs`) found in `WPThemeReview/Tests/CoreFunctionality/FileIncludeSniff.inc`.
+If you run
+
+```bash
+$ cd /path-to-cloned/WPThemeReview
+$ vendor/bin/phpcs --standard=WPThemeReview -s WPThemeReview/Tests/CoreFunctionality/FileIncludeUnitTest.inc --sniffs=WPThemeReview.CoreFunctionality.FileInclude
+```
+
+The results will be
+
+```bash
+...
+FILE: /Users/infinum-denis/Projects/Personal/WPThemeReview/WPThemeReview/Tests/CoreFunctionality/FileIncludeUnitTest.inc
+--------------------------------------------------------------------------------
+FOUND 0 ERRORS AND 6 WARNINGS AFFECTING 6 LINES
+--------------------------------------------------------------------------------
+ 3 | WARNING | Check that include is not being used to load template
+   |         | files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+ 4 | WARNING | Check that include_once is not being used to load
+   |         | template files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+ 5 | WARNING | Check that require is not being used to load template
+   |         | files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+ 6 | WARNING | Check that require_once is not being used to load
+   |         | template files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+ 7 | WARNING | Check that include is not being used to load template
+   |         | files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+ 8 | WARNING | Check that require_once is not being used to load
+   |         | template files. "get_template_part()" should be used to load template files.
+   |         | (WPThemeReview.CoreFunctionality.FileInclude.FileIncludeFound)
+--------------------------------------------------------------------------------
+
+Time: 45ms; Memory: 6Mb
+```
+
+You'll see the line number and number of WARNINGs we need to return in the `getWarningList()` method.
+
+The `--sniffs=...` directive limits the output to the sniff you are testing.
+
+You can also run the same command using composer scripts
+
+```bash
+composer check-cs -- --standard=WPThemeReview -s WPThemeReview/Tests/CoreFunctionality/FileIncludeUnitTest.inc --sniffs=WPThemeReview.CoreFunctionality.FileInclude
+```
+
+## Code Standards for this project
```

Comment:
It may not be a bad idea to move this up and make this a subsection of `Contributing patches and new features`.